src/Microsoft.Tye.Hosting/TransformProjectsIntoContainers.cs

```diff
@@ -41,16 +41,17 @@ private async Task TransformProjectToContainer(Model.Application application, Mo
             var serviceDescription = service.Description;
             var serviceName = serviceDescription.Name;
 
-            var expandedProject = Environment.ExpandEnvironmentVariables(project.Project);
-            var fullProjectPath = Path.GetFullPath(Path.Combine(application.ContextDirectory, expandedProject));
-            service.Status.ProjectFilePath = fullProjectPath;
+            service.Status.ProjectFilePath = project.ProjectFile.FullName;
+            var targetFramework = project.TargetFramework;
```

Comment:
It looks like TargetFramework can be null if TargetFrameworks isn't null. Should we assume one of the target frameworks as we are publishing on behalf of the person doing tye run?